### PR TITLE
add cache_images to lightning

### PR DESCRIPTION
**Pull request by dberenbaum**
Adds `dvclive.lightning.DVCLiveLogger(cache_images=True)`

### CODE REVIEW

File: src/dvclive/lightning.py [hunk 1]
```diff
@@ -44,6 +44,7 @@ def __init__(
         report: Optional[str] = "auto",
         save_dvc_exp: bool = False,
         dvcyaml: bool = True,
+        cache_images: bool = False,
```
Comment (daavoo) [inline src/dvclive/lightning.py:47]: You need to also add it to the `_live_init` dict below.

**Review by daavoo — CHANGES_REQUESTED** [review]

**Comment by dberenbaum** [comment]
@daavoo Any ideas for how to test when adding a new `Live()` option that it also exists in the lightning logger?

**Comment by daavoo** [comment]
> @daavoo Any ideas for how to test when adding a new `Live()` option that it also exists in the lightning logger?

I guess we need to remember to update this test with the new args: https://github.com/iterative/dvclive/blob/a0de508ace2df86413b3d9952abaa652cbbdad5d/tests/test_frameworks/test_lightning.py#L144-L158

Seems like the reasonable thing to do since we don't change or have that many args and it is the only framework where the args are not directly passed like in other frameworks as `Live(**kwargs)`

**Comment by dberenbaum** [comment]
> Seems like the reasonable thing to do since we don't change or have that many args and it is the only framework where the args are not directly passed like in other frameworks as `Live(**kwargs)`

Added a test, but I think not changing it often is why I forget to do it. Anyways, I don't have any better ideas for now. Just a thought.

**Review by daavoo — APPROVED** [review]

**Pull request merged**